com/ideas2it/employee/dao/impl/EmployeeDao.java

```diff
@@ -65,6 +69,7 @@ public boolean addEmployee(Employee employee) throws EMSException {
             isAdded = addAddress(employee.getAddress(), employeeId);
 
         } catch (SQLException e) {
```

Comment:
Can you debug what is the actual error with this log? Add actual exception in this log